framework/include/gvk.hpp

```diff
@@ -104,6 +106,7 @@
 #include <avk/avk.hpp>
 
 // -------------------- Gears-Vk includes --------------------
+#include "format_for.hpp"
```

Comment:
Okay, that's fine. I wonder what changed that made this necessary.

Reply:
Yes, I'd like to know that too. I'll check the MSVC version on my old computer and investigate.